Lombiq.Tests.UI/Services/WebDriverFactory.cs

```diff
@@ -129,36 +141,39 @@ private static TDriver SetCommonTimeouts<TDriver>(this TDriver driver, TimeSpan
     }
 
     private static TDriver CreateDriver<TDriver>(Func<TDriver> driverFactory, IDriverConfig driverConfig)
```

Comment:
I've allowed myself for some refactoring of this methods which leads to more readable code. See my PR: https://github.com/Lombiq/UI-Testing-Toolbox/pull/167. All UI tests passed successfully locally.